File: src/tutorials/code/node/rest/2.js
'use strict';

const Mixer = require('beam-client-node');

const client = new Mixer.Client(new Mixer.DefaultRequestRunner());

const channelName = process.argv[2];

client.request('GET', `channels/${channelName}`)
.then(res => {
    const viewers = res.body.viewersTotal;
    console.log(`You have ${viewers} total viewers...`);
});

File: src/tutorials/code/python/rest/2.py
import requests
import sys

s = requests.Session()
s.headers.update({'Client-ID': 'CLIENT_ID'})

channel_response = s.get('https://mixer.com/api/v1/channels/{}'.format(sys.argv[1]))

viewers = channel_response.json()['viewersTotal']
print("You have {} viewers...".format(viewers))
